src/oom/meson.build

```diff
@@ -3,6 +3,9 @@
 systemd_oomd_sources = files('''
         oomd-util.c
         oomd-util.h
+        oomd-manager.c
+        oomd-manager.h
+        oomd.c
```

Comment:
doesn't matter, but we try to order files alphabetically in the build system too